src/Integration.UnitTests/Suppression/SonarQubeIssuesProviderTests.cs

```diff
@@ -105,7 +111,7 @@ public void Constructor_TimerIsInitialized()
             // Arrange
             // The provider will attempt to fetch issues at start up and will loop if it
             // is not connected to the server, so we'll initialise it as connected.
-            SetupSolutionBinding(isConnected: true, issues: null);
```

Comment:
Miscellaneous test fix: passing `issues: null` causes the product code to throw an exception, which is caught and handled internally.

However, it isn't what the tests are expecting i.e. the tests are passing but perhaps not for the expected reason. I've modified the `SetupSolutionBinding` method to use an empty list of issues instead.